README.md

```diff
@@ -150,6 +150,56 @@ co(function * () {
 
 ```
 
+### Shared Secret Example
+
+Sometimes it is desirable that the sender can choose the amount and generate the
+condition without communicating with the recipient. This is an example of a
+payment using the Shared Secret Protocol (SSP) which implements this type of
+flow.
+
+```js
+'use strict'
+
+const co = require('co')
+const ILP = require('.')
+const FiveBellsLedgerPlugin = require('ilp-plugin-bells')
+
+const sender = ILP.createSender({
+  _plugin: FiveBellsLedgerPlugin,
+  account: 'https://localhost/ledger/accounts/alice',
+  password: 'alice'
+})
+
+const receiver = ILP.createReceiver({
+  _plugin: FiveBellsLedgerPlugin,
+  account: 'https://localhost/ledger/accounts/bob',
+  password: 'bobbob'
+})
+
+co(function * () {
+  yield receiver.listen()
+  receiver.on('incoming', (transfer, fulfillment) => {
```

Comment:
Is the receiver going to fulfill everything that comes in? Wouldn't you want a callback function or something to let the application check whether they want the money?